documents/ANDROID.md

```diff
@@ -37,12 +37,12 @@ a dependency of the aws-iot-device-sdk-android library.
 * Java 11+ ([Download and Install Java](https://www.java.com/en/download/help/download_options.html))
   * [Set JAVA_HOME](./PREREQUISITES.md#set-java_home)
 * Gradle 7.4.2 ([Download and Install Gradle](https://gradle.org/install/))
-* Android SDK 26 ([Doanload SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
+* Android SDK 24 ([Download SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
   * [Set ANDROID_HOME](./PREREQUISITES.md#set-android_home)
+##### NOTE: The SDK supports Android minimum API of 24 but requires [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) to support Java 8 language APIs used in by the SDK. If minimum Android API Version is set to 26+ desugaring is not required.
```

Comment:
Debatable and trivial and nit: github markdown syntax supports alerts: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

May be worth using a NOTE alert here.

Reply:
cool!